admin_guide/allocating_node_resources.adoc

```diff
@@ -0,0 +1,115 @@
+= Allocating Node Resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node can reserve a portion of its resources for use by all underlying
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[node
+components] (e.g., kubelet, kube-proxy, Docker) and the remaining system
```

Comment:
I agree, for the sake of consistency.

However, if we use "node" for both, then we need to find another way to distinguish OpenShift from non-OpenShift components.  Unfortunately, something like "...use by all underlying OpenShift components (...) and non-OpenShift components (...)" sounds a bit awkward.